src/roman_datamodels/maker_utils/_datamodels.py

```diff
@@ -174,6 +174,7 @@ def mk_level3_mosaic(*, shape=(4088, 4088), n_images=2, filepath=None, **kwargs)
 
     wfi_mosaic = stnode.WfiMosaic()
     wfi_mosaic["meta"] = mk_mosaic_meta(**kwargs.get("meta", {}))
+    # wfi_mosaic["meta"]['basic'] = mk_mosaic_basic(**kwargs.get("meta", {}))
```

Comment:
```suggestion
    wfi_mosaic["meta"]['basic'] = mk_mosaic_basic(**kwargs.get("meta", {}))
```

Mosaic basic still exists.. it just lost some members.